docs/entity/validators.md

```diff
@@ -29,26 +29,25 @@ See the following example usage:
 
 ```json
 {
-    "type": "regex",
-    "errorMsg": "Input Name must begin with a letter and consist exclusively of alphanumeric characters and underscores.",
-    "pattern": "^[a-zA-Z]\\w*$"
+  "type": "regex",
+  "errorMsg": "Input Name must begin with a letter and consist exclusively of alphanumeric characters and underscores.",
+  "pattern": "^[a-zA-Z]\\w*$"
 }
 ```
 
 ### Number
 
 <h3> Properties </h3>
 - `range`<span class="required-asterisk">*</span> is the range within which the target value should fall.
+- `isInteger` is the boolean which target only integer value if sets true. Default value is false
```

Comment:
how it would behave if I put 1024.0?

Reply:
it will not raise any error, as according to the code, we first parsed the value into a `Number` so it will convert it into `1024` and treat it as an integer.